docs/Using Price Feeds/feed-registry.html

```diff
@@ -69,13 +66,8 @@
   address public constant USD = address(840);
   address public constant GBP = address(826);
   address public constant EUR = address(978);
-  address public constant JPY = address(392);
-  address public constant KRW = address(410);
-  address public constant CNY = address(156);
-  address public constant AUD = address(36);
-  address public constant CAD = address(124);
-  address public constant CHF = address(756);
-  address public constant ARS = address(32);
+  
+  // ... other fiat currencies
 }
 ```
 ```javascript Solidity 7
```

Comment:
Do we need Solidity 7? As far as I know, it's a skippable version and most people use either v0.6 or v0.8.

And do we really need to maintain the same code for multiple solidity versions? After all of the Chainlink contracts are moved to v0.8, we plan to maintain the latest version only for all of our products. I think we have to follow the same approach here? Thoughts?

Reply:
Sure, we can just keep Solidity 8.